src/main/webapp/script.js

```diff
@@ -56,14 +82,84 @@ function passQuery() {
     });
 }
 
+function displayAmChartsMap(data, stateName) {
+  am4core.useTheme(am4themes_animated);
+  const chart = am4core.create('map', am4maps.MapChart);
+  chart.height = 550;
+
+  // Create map instance
+  chart.geodata = states[stateName].geoData;
+  chart.projection = new am4maps.projections.AlbersUsa();
+  const polygonSeries = chart.series.push(new am4maps.MapPolygonSeries());
+
+  // Set min/max fill color for each area
+  polygonSeries.heatRules.push({
+    property: 'fill',
+    target: polygonSeries.mapPolygons.template,
+    min: chart.colors.getIndex(1).brighten(1),
+    max: chart.colors.getIndex(1).brighten(-0.3),
+    logarithmic: true,
+  });
+  polygonSeries.useGeodata = true;
+
+  const countyData = data.map((value) => {
+      return {id: value[2] + value[3], value: value[1]};
+  });
+  polygonSeries.data = countyData;
+
+  // Set up heat legend
```

Comment:
This could be something for later, but the GeoCharts heat legend and the AmCharts heat legend look very similar but are positioned very differently on the page. It would be nice to have them appearing in the same spot.